packages/components/src/components/progress/test/snapshot.spec.tsx

```diff
@@ -11,9 +11,7 @@ const DEFAULT_HTML_PROGRESS_PROPS = `aria-busy="true" max="${DEFAULT_PROPS._max}
 const DEFAULT_HTML_SPAN_PROPS = `aria-live="polite" aria-relevant="removals text"`;
 
 describe('test Progress', () => {
-	beforeAll(() => {
-		jest.useFakeTimers();
-	});
+	beforeAll(() => {});
```

Comment:
Kann das nicht ganz raus?
Vermutlich können wir hier auch mal auf Snapshots umstellen, muss aber nicht in diesem PR passieren. Ist wohl ein Spezialfall.